framework/src/systems/NonlinearSystemBase.C

```diff
@@ -3176,6 +3176,14 @@ NonlinearSystemBase::computeScalingJacobian(NonlinearImplicitSystem & sys)
     TIME_SECTION(_compute_scaling_jacobian_timer);
 
     auto & petsc_matrix = *static_cast<PetscMatrix<Real> *>(sys.matrix);
+
+    PetscInt m, n;
+    auto ierr = MatGetLocalSize(petsc_matrix.mat(), &m, &n);
```

Comment:
ierr = MatGetLocalSize(petsc_matrix.mat(), &m, NULL);

If we do not care about m, just ignore it.